### PR TITLE
Updates to reflect the current status of the system.

### PR DESCRIPTION
**Pull request by ryanbnl**
Minor update to reflect the actual state of the system.

### CODE REVIEW

**Comment by ryanbnl** [comment]
Initial pass to bring the document up-to-date. It probably requires another pass later on to add any missing but important information.

File: README.md [hunk 1]
```diff
@@ -39,19 +39,18 @@ The following requirements and principles have been used when designing the Elec
...
-
-involved in the validation process in the verifier. If a CSCA'ss public keys appear in the list - they are _only_ there to facilitate the creation of the trusted list of public keys itself. They are not used during verification of an HCERT (as this is generally offline -- and purely based on the trusted list of that day).
+The trusted keys which will be used by verifiers are published in a list which includes all public keys together with issuer metadata. The keys which from time to time are used to sign the HCERTs and should be trusted are included on the Trusted List. There are no CAs or other intermediate parties involved in the validation process in the verifier. If a CSCA'ss public keys appear in the list - they are _only_ there to facilitate the creation of the trusted list of public keys itself. They are not used during verification of an HCERT (as this is generally offline - and purely based on the trusted list of that day).
 
+Revocation is implemented via omission - t
```
Comment (dslmeinte) [inline README.md:48]: Unfinished sentence?

File: README.md [hunk 2]
```diff
@@ -39,19 +39,18 @@ The following requirements and principles have been used when designing the Elec
...
 
 ## Known Implementations
 
 Multiple implementations are available via the ["European eHealth network - digital green development coordination"
-](https://github.com/ehn-dcc-development) GitHub repository.
+](https://github.com/ehn-dcc-development) GitHub repository.
```
Comment (dslmeinte) [inline README.md:53]: (extra space?)

File: hcert_spec.md [hunk 3]
```diff
@@ -274,18 +270,19 @@ In a later version, the service may also:
 
 The tasks that are marked _DCCG_ or _CIRCABC_ are expected to be handled by DCCG automation, CIRCABC or similar systems under control and responsibility of the Secretariat.
 
-The format for the lists used for the interchange between the Member States and the Secretariat is waiting for the completion of the T-Systems/SAP proposal -- and should be optimised for clarity and interoperability. The ICAO Master List structure as defined in Doc 9303 part 12 may be considered.
+The format for the lists used for the interchange between the Member States and the Secretariat is available in DGC Gateway's github repository [here](https://github.com/eu-digital-green-certificates/dgc-gateway). The specification is distributed as a build artifact, the latest of which (at time of writing) can be found [here](https://github.com/eu-digital-green-certificates/dgc-gateway/releases/download/1.3.21/openapi.json)
```
Comment (skounis) [inline hcert_spec.md:273]: I prefer avoiding the work `here` for making links. I always prefer to make the links parts of the sentence. 
```markdown
available in [DGC Gateway's github repository](https://github.com/eu-digital-green-certificates/dgc-gateway)
```

File: README.md [hunk 4]
```diff
@@ -39,19 +39,18 @@ The following requirements and principles have been used when designing the Elec
...
-
-involved in the validation process in the verifier. If a CSCA'ss public keys appear in the list - they are _only_ there to facilitate the creation of the trusted list of public keys itself. They are not used during verification of an HCERT (as this is generally offline -- and purely based on the trusted list of that day).
+The trusted keys which will be used by verifiers are published in a list which includes all public keys together with issuer metadata. The keys which from time to time are used to sign the HCERTs and should be trusted are included on the Trusted List. There are no CAs or other intermediate parties involved in the validation process in the verifier. If a CSCA'ss public keys appear in the list - they are _only_ there to facilitate the creation of the trusted list of public keys itself. They are not used during verification of an HCERT (as this is generally offline - and purely based on the trusted list of that day).
```
Comment (dslmeinte) [inline README.md:46]: Twice replace " - " (last 2 occurrences) &rarr; ", "?

File: README.md [hunk 5]
```diff
@@ -39,19 +39,18 @@ The following requirements and principles have been used when designing the Elec
 
 ## Trust model
 
-The trust model is currently under development. It is assumed to be a health-specific version of the ICAO Master List concept (see also https://www.who.int/publications/m/item/interim-guidance-for-developing-a-smart-vaccination-certificate) that is both health and COVID-19 specific.
+The trust model is based on the ICAO Master List concept with some minor modifications to improve performance and compatibility. At the center of the model is the [DGC Gateway](https://github.com/eu-digital-green-certificates/dgc-gateway) and the associated [certificate governance](https://github.com/eu-digital-green-certificates/dgc-overview/blob/main/guides/certificate-governance.md). The gateway operates in a B2B model with onboarded countries. Member States are then responsible for publishing their own trust lists publicly on their own national backends.
```
Comment (dslmeinte) [inline README.md:42]: All occurrences of "DGC" could be replaced by "DCC", since the GitHub repo links are also renamed (but better check).

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (skounis) [reply README.md:42]: @dslmeinte GitHub link/repositories were not renamed 

We should stay with https://github.com/eu-digital-green-certificates/dgc-gateway. The [/gcc-gateway](https://github.com/eu-digital-green-certificates/dcc-gateway) gives `404`

Comment (dslmeinte) [reply README.md:42]: Oh yeah, not our repos...

OK, then at least the link _text_ should say "DCC" wherever it makes sense.

Comment (ryanbnl) [reply README.md:41]: The test should match the name of the repository it's pointing to and unfortunately that still has the old name. I can understand why - we still have issues due to the renaming of the ehn organisation.

**Review by dslmeinte — DISMISSED** [review]

**Comment by ryanbnl** [comment]
Now I'm done, a last review / approval would be appreciated @dslmeinte @skounis :)

**Review by dslmeinte — APPROVED** [review]

**Review by skounis — APPROVED** [review]

**Pull request merged**